DasharoPayloadPkg/Library/FmpDeviceSmmLib/FmpDeviceSmmLib.c

```diff
@@ -9,10 +9,12 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 **/
 
-#include <PiDxe.h>
+#include <Coreboot.h>
 #include <Guid/SystemResourceTable.h>
 #include <Library/BaseMemoryLib.h>
 #include <Library/BlParseLib.h>
+#include <Library/BootLogoLib.h>

```

Comment:
I think this library is not actually used but gets added here and in the `.inf`-file.